### DISCUSSION TITLE
posts/when-the-world-met-me/

### DISCUSSION THREAD

**Discussion by giscus** [WincerChan/blog-comments#7] [Announcements]
# posts/when-the-world-met-me/

我其实之前一直对旅行提不起什么兴趣，一开始觉得这只是因为我的性格比较内向、不愿同陌生人发生不必要接触，后来往深了想这其实是来自于我内心追求安逸、拒绝踏出舒适圈的映射。那么为什么我会想到来突破这个舒适圈呢，这很大程度归咎于我开始健身了。从三个月前跑两公里就累得要死，到现在已经可以比较轻松地跑完五公里，我的确成长了不少。因此我觉得适当突破一下舒适圈也许不是什么坏事情，于是就有了这次的韩国旅行。

https://blog.itswincer.com/posts/when-the-world-met-me/

**Comment by yzyyz1387** [comment]
羡慕住了，我觉得无论最终对旅行的体验如何，只要是跟对象一起度过的，都挺有意义，以后想起来也不错。
另外，来点照片看看红鼻子，哈哈哈哈哈
好久不来，先祝你端午安康咯

**Reply by WincerChan** [reply]
照片就不发了哈哈哈，有点滑稽。
好久不见，也祝你端午安康
